Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+**Usability Features:**
+
+* Import via QR or a variety of text specifications.
+* Export via QR or Clipboard or printing.
+* Identify Seeds using [Object Identity Block](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2021-002-digest.md#object-identity-block).
+ 
+**Security Features:**
+
+* Stored Securely with Apple Encryption.
+* Protected via 2FA: you must login in to your Apple account, then you must verify whenever you access private data.
+* Automated iCloud backup and recovery.
+
+**Gordian Seed Tool** is a reference app, demonstrating the [Gordian Principles](https://github.com/BlockchainCommons/GordianSeedTool-iOS#gordian-principles) of independence, privacy, resilience, and openness.
+
+## Installing Seed Tool
+
+You can either purchase **Gordian Seed Tool** from the Apple store or you can compile from the source here.
+
+For full functionality of the iCloud backup, be sure to turn on the following functionality under "Settings > Apple ID > iCloud" on all devices running **Gordian Seed Tool**:
+
+* Keychain
+* iCloud Drive
+
+Be _very_ sure that all devices running **Gordian Seed Tool** are fully logged into your Apple account, with full access to iCloud, and switches set as noted above. Failure to do so will result in seed entries not being synced to the iCloud (or other devices).
+
+## Using Seed Tool
+
+**Gordian Seed Tool** is a storage mechanism for seeds, particularly those used in cryptography systems. Seeds used in it will usually follow a three-part cycle.
+
+1. **Add a Seed.** First, you must add seeds to the system. There are two ways to do so.
+   * **Import a Seed.** You can import an existing seed that you generated in a cryptowallet.
+   * **Create a Seed.** You can create a new seed.
+2. **Store a Seed.** Your seed will be encrypted (and also backed up if you have iCloud enabled).
+   * **View & Edit a Seed.** While you are storing a seed, you will be able to view it and change its metadata.
+   * **Read an OIB.** Each seed (and key) comes with an Identity Block that makes it easy to identify.
+3. **Use a Seed.** You can actively use a seed that is stored in **Gordian Seed Tool** without ever having to export it. 
+   * **Answer Requests.** Seed Tool uses the [`crypto-request`/`crypto-response`](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2021-001-request.md) system defined by Blockchain Commons for URs. This allows Seed Tool to export precisely what's needed by another app.
+   * **Derive a Key.** Alternatively, you can choose to export specific derived keys on your own, while keeping the seed in the app.
+   * **Shard a Seed.** Finally, you can improve the resilience of your seed by sharding it with SSKR and giving out those shares.
```

Comment:
I wonder if shard a seed should be a sub-item of #2, which is to shard a seed to store a seed redundantly offline to different locations or help from your close family, friends or colleagues.

Reply:
I considered the possibilities, and I think this is the best option. Putting Shard a Seed in (2) Store a Seed would be confusing because it's really usage, and moving it to an Export/Shard section, though thematically at least as good as keeping it in Use, would decrease the prominence of sharding.